src/components/GridTemplates/DetailsPage1/_index.scss

```diff
@@ -0,0 +1,35 @@
+////
+/// Details page template 1 styles.
+/// @group accordion
+/// @copyright IBM Security 2018
+////
+
+@import '../../../globals/namespace/index';
+@import '../../../globals/layout/index';
+
+@include export-namespace($name: 'details-page-1') {
```

Comment:
What's the rationale behind using `export-namespace($name: details-page-1)` over `@include security--component($name: details-page-1)` here?

Reply:
If I remember correctly, I just copied this scss file from the `Accordion` to use as a template, so that's why it has `export-namespace` & the wrong namespace in the top-of-file comments. 

Fixed now 👍